### PR TITLE
Add crossmark symbol sym.rs

### PR DESCRIPTION
**Pull request by giannissc**

### CODE REVIEW

**Comment by giannissc** [comment]
Ready! 🙂 @Enivex

File: crates/typst/src/symbols/sym.rs [hunk 1]
```diff
@@ -464,7 +464,13 @@ pub(crate) const SYM: &[(&str, Symbol)] = symbols! {
 
     // Miscellaneous.
     ballot: ['☐', x: '☒'],
-    checkmark: ['✓', light: '🗸'],
+    mark:[
+        '✓',
```
Comment (Enivex) [inline crates/typst/src/symbols/sym.rs:468]: This should presumably be `check: '✓',`

Comment (giannissc) [reply crates/typst/src/symbols/sym.rs:468]: Is it ok not to have a default?

Comment (Enivex) [reply crates/typst/src/symbols/sym.rs]: > Is it ok not to have a default?

There are plenty of examples of this in the file :)

**Comment by giannissc** [comment]
Ready!

File: crates/typst/src/symbols/sym.rs [hunk 2]
```diff
@@ -464,7 +464,13 @@ pub(crate) const SYM: &[(&str, Symbol)] = symbols! {
 
     // Miscellaneous.
     ballot: ['☐', x: '☒'],
-    checkmark: ['✓', light: '🗸'],
+    mark:[
```
Comment (Coekjan) [inline crates/typst/src/symbols/sym.rs:467]:
```suggestion
    mark: [
```

Comment (giannissc) [reply crates/typst/src/symbols/sym.rs]: Rebased and Done!

**Comment by Coekjan** [comment]
Looks good to me

**Review by Coekjan — COMMENTED** [review]
Looks good to me.

> I am not the author/collaborator of the repo, so you still need to wait for their reviews.

**Comment by laurmaedje** [comment]
@Enivex there is also `refmark` and `servicemark` further below, so changing `checkmark` to `mark.check` maybe makes that a little inconsistent? It's probably also a bit harder to find `mark.check` than `checkmark`.

**Comment by Enivex** [comment]
> @Enivex there is also `refmark` and `servicemark` further below, so changing `checkmark` to `mark.check` maybe makes that a little inconsistent? It's probably also a bit harder to find `mark.check` than `checkmark`.

I don't think so. Reference mark and service mark seem to fulfil entirely different roles than a check mark, so I'm not sure they belong together. On the other hand, a cross mark and check mark definitely do.

I think most people should just search for "check" or "mark" anyway.

**Comment by Enivex** [comment]
> I didn't mean to imply that they belong together, more that `mark` is maybe a little to general as a parent for `mark.check` since there are so many other kinds of marks. So maybe `checkmark` and `crossmark` would be more consistent.

They could be grouped under ballot instead?

**Comment by laurmaedje** [comment]
The existing `ballot` symbols all have the box as part of the glyph, so I think that would also be a bit inconsistent. Naming symbols is really tricky ^^

**Comment by Enivex** [comment]
> The existing `ballot` symbols all have the box as part of the glyph, so I think that would also be a bit inconsistent. Naming symbols is really tricky ^^ 

On the other hand, the Unicode name for the cross mark is `ballot x` 😛.

Feel free to do what you think is right. (It could always be changed later too.)

**Comment by laurmaedje** [comment]
I think for the time being, I'd prefer the original `checkmark` / `crossmark`. `lt` and `gt` are also not grouped under a shared parent, which is sort of similar I guess.

**Comment by laurmaedje** [comment]
Thanks!

**Comment by giannissc** [comment]
Let me know if you would like any other changes! @laurmaedje , @Enivex

**Comment by laurmaedje** [comment]
No, it's all good now. It will be merged automatically once CI has passed. :)

**Pull request merged**